src/Metadata/Resource/Factory/ResourceMetadataFactoryTrait.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace ApiPlatform\Core\Metadata\Resource\Factory;
+
+use ApiPlatform\Core\Exception\ResourceClassNotFoundException;
+use ApiPlatform\Core\Metadata\Resource\Operation;
```

Comment:
That's incorrect. `OperationResourceMetadataFactory` only provides the
defaults.
On 6 Apr 2016 10:17 p.m., "Antoine Bluchet" <notifications@github.com>
wrote:

> In src/Metadata/Resource/Factory/ResourceMetadataFactoryTrait.php
> <https://github.com/api-platform/core/pull/490#discussion_r58712609>:
>
> > @@ -0,0 +1,124 @@
> > +<?php
> > +
> > +namespace ApiPlatform\Core\Metadata\Resource\Factory;
> > +
> > +use ApiPlatform\Core\Exception\ResourceClassNotFoundException;
> > +use ApiPlatform\Core\Metadata\Resource\Operation;
>
> Actually we should be able to remove this whole portion of code
> <https://github.com/api-platform/core/blob/a7441dfcf2349eb94cd590541409b5095efb51fd/src/Metadata/Resource/Factory/AnnotationResourceMetadataFactory.php#L102-L143>.
> Operations are handled through OperationResourceMetadataFactory
> <https://github.com/api-platform/core/blob/a7441dfcf2349eb94cd590541409b5095efb51fd/src/Metadata/Resource/Factory/OperationResourceMetadataFactory.php>
> .
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/api-platform/core/pull/490/files/37f58321cc8415e7f464031ee8431b73aa20417d#r58712609>
>